exasol_integration_test_docker_environment/lib/test_environment/database_waiters/is_database_ready_thread.py

```diff
@@ -44,14 +46,16 @@ def run(self):
     def create_db_connection_command(self):
         username = self.database_credentials.db_user
         password = self.database_credentials.db_password
-        connection_options = f"""-c '{self._database_info.host}:{self._database_info.db_port}' -u '{username}' -p '{password}'"""
-        cmd = f"""$EXAPLUS {connection_options}  -sql 'select 1;' -jdbcparam 'validateservercertificate=0'"""
+        connection_options = f"""-c 'localhost:{self._database_info.db_port}' -u '{username}' -p '{password}'"""
+
+        exaplus = f"/usr/opt/EXASuite-7/EXASolution-{self.docker_db_image_version}/bin/Console/exaplus"
```

Comment:
This does not work for db image: `7.1.0-d1` as there the path is:
`/usr/opt/EXASuite-7/EXASolution-7.1.0/bin/Console/exaplus`
and not
`/usr/opt/EXASuite-7/EXASolution-7.1.0-d1/bin/Console/exaplus`